ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -254,3 +255,33 @@ def test_numpy_rfftn(dtype_and_x, frontend, backend_fw, test_flags, fn_tree, on_
         axes=axes,
         norm=norm,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.ifft2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"), 
+        shape=(4, 4), array_api_dtypes=True
+    )
+)    
+def test_numpy_ifft2(
+    dtype_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        backend_to_test=backend_fw,
```

Comment:
Please try to run the tests locally and check the logs, "backend_fw" is not defined

Reply:
Hello @AlexandraPosta! I have added the "backend_fw". However, when I try to run the test I get this ImportError;

"ImportError: cannot import name 'array_helpers_dtype_info_helper' from partially initialized module 'ivy_tests.test_ivy.helpers.hypothesis_helpers.array_helpers' (most likely due to a circular import) (/home/codespace/.local/lib/python3.10/site-packages/ivy-1.1.9-py3.10.egg/ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py)"

![Ivy error](https://github.com/unifyai/ivy/assets/73750165/2a4bbee3-dbdc-40da-b498-e4a7fa3c431a)